restcomm/restcomm.testsuite/src/test/java/org/restcomm/connect/testsuite/http/RecordingEndpointTest.java

```diff
@@ -128,6 +128,35 @@ public void getRecordingListFilteredByCallSid() {
         assertTrue(filteredRecordingsByCallSid.get("end").getAsInt() == 17);
     }
 
+    @Test
+    public void deleteRecording() {
+        JsonObject firstPage = RecordingEndpointTool.getInstance().getRecordingList(deploymentUrl.toString(), adminAccountSid,
```

Comment:
i think this first part about asserting the inital size of the list is not neccesary

Reply:
@jaimecasero ok I can remove this